package.json

```diff
@@ -76,6 +76,7 @@
     "session-file-store": "1.5.0",
     "sharp": "0.32.6",
     "swagger-ui-express": "4.3.0",
+    "type-fest": "^4.26.0",
```

Comment:
the other deps are fixed version, this has an ^.  Is there a specific reason for this?

Reply:
it will be removed in a later PR
it was a test to type some complex types, but it was just easier to make a new enum for it